scipy/optimize/optimize.py

```diff
@@ -441,7 +441,8 @@ def function_wrapper(*wrapper_args):
 
 
 def fmin(func, x0, args=(), xtol=1e-4, ftol=1e-4, maxiter=None, maxfun=None,
-         full_output=0, disp=1, retall=0, callback=None, initial_simplex=None):
+         full_output=0, disp=1, retall=0, callback=None, initial_simplex=None,
```

Comment:
We recommend that the `minimize` function is used for new code, `fmin` is retained for 'legacy' purposes. It's fine to add adaptive here.